Dockerfiles/manifests/all-containers/system-probe-configmap.yaml

```diff
@@ -16,7 +16,7 @@ data:
       enable_tcp_queue_length: false
       enable_oom_kill: false
       collect_dns_stats: false
-      max_tracked_connections: 65536
+      max_tracked_connections: 131072
```

Comment:
This file is generated so that this change will be overridden at the next generation.
This file generated from https://github.com/DataDog/helm-charts/blob/datadog-2.10.4/charts/datadog/templates/system-probe-configmap.yaml#L37
where the actual value comes from https://github.com/DataDog/helm-charts/blob/datadog-2.10.4/charts/datadog/values.yaml#L319

So, we should do a PR on https://github.com/DataDog/helm-charts/blob/datadog-2.10.4/charts/datadog/values.yaml#L319 to persist this change.

Reply:
=> https://github.com/DataDog/helm-charts/pull/199